src/it/scala/io/iohk/ethereum/txExecTest/util/DumpChainApp.scala

```diff
@@ -73,10 +74,10 @@ object DumpChainApp extends App with NodeKeyBuilder with SecureRandomBuilder wit
     val peerMessageBus = actorSystem.actorOf(PeerEventBusActor.props)
 
     val peerManager = actorSystem.actorOf(PeerManagerActor.props(
+      peerDiscoveryManager = actorSystem.deadLetters, // TODO: fixme
```

Comment:
Sounds good. I wouldn't like to spend time on fixing it now, if we're not sure anyone is going to use it again.